.yarn/versions/ca30052e.yml

```diff
@@ -0,0 +1,31 @@
+releases:
```

Comment:
Only need a patch release on `@yarnpkg/cli` and `@yarnpkg/plugin-compat`, the rest should be declined

Reply:
Oh, ok. This is the first time I'm using the version plugin.

Reply:
Fixed